src/service/url-replacements-impl.js

```diff
@@ -658,9 +673,7 @@ export class GlobalVariableSource extends VariableSource {
   getVariantsValue_(getter, expr) {
     return Services.variantsForDocOrNull(this.ampdoc.getHeadNode())
         .then(variants => {
-          userAssert(variants,
-              'To use variable %s, amp-experiment should be configured',
-              expr);
+          userAssertExtensionConfigured(variants, expr, 'amp-experiment');
```

Comment:
👍

Reply:
Meaning "ditto, bad size win", or "good"?

Reply:
This is fine.